torchaudio/prototype/models/hdemucs.py

```diff
@@ -929,3 +929,48 @@ def _ispectro(z: torch.Tensor, hop_length: int = 0, length: int = 0, pad: int =
     _, length = x.shape
     other.append(length)
     return x.view(other)
+
+
+def hdemucs_low(sources: List[str], sample_rate: int) -> HDemucs:
+    r"""Builds low nfft version of HDemucs model.
```

Comment:
`low nfft version of` is not quite helpful. Can we either have more detailed description or say something like "the version referred to X in the paper".